templates/bareos-dir/pool/pool.conf.j2

```diff
@@ -6,6 +6,8 @@ Pool {
   AutoPrune = yes                     # Prune expired volumes
   Volume Retention = {{ item.retention }}          # How long should the Incremental Backups be kept?  (#12)
   Maximum Volume Bytes = {{ item.max_vol_bytes }}           # Limit Volume size to something reasonable
-  Maximum Volumes = {{ item.max_vol }}               # Limit number of Volumes in Pool  
+{% if item.max_vol is defined %}
```

Comment:
Should a similar condition apply to other variables used in this template? (`item.retention`, `item.max_vol_bytes`, etc)

Reply:
The doc doesn't clearly states which vars are optional. I just added item.max_vol_bytes which seems to be optional according to https://docs.bareos.org/TasksAndConcepts/VolumeManagement.html